hip_data_tools/etl/athena_to_adwords.py

```diff
@@ -123,8 +123,14 @@ def _upload_conversions(self, data_batch):
         return self._get_adwords_util().upload_conversions(data_batch)
 
     def _state_manager_connect(self):
+
+        LOG.debug("Connecting to Cassandra")
+
         conn = CassandraConnectionManager(self.__settings.etl_state_manager_connection)
         conn.setup_connection(self.__settings.etl_state_manager_keyspace)
+
+        LOG.debug(f"Cassandra connection established: {conn.get_cluster()}")
```

Comment:
Should this maybe be `Log.info`. It's usually only done once at the beginning. @datascienceChris